### ISSUE TITLE
text in README.md about makefiles

### ISSUE THREAD

**Issue by cperciva** [Tarsnap/spiped#40]
```
Official releases on www.tarsnap.com/spiped.html have POSIX-compliant Makefiles; please use those.

:warning: The git repository (including the tag-based "release" snapshots on github) uses BSD Makefiles, which may or may not work on your operating system.
```

These comments have been out of date since `c46ceda5` I think?

**Comment by gperciva** [comment]
Hmm.  Those comments were added in 29fe39b2 2015-10-21, whereas c46ceda5 was 2015-05-26.  We added this warning specifically in response to https://github.com/Tarsnap/spiped/issues/26 .

That said, the complaint was about spiped 1.5.0, which came before c46ceda5.  So perhaps we can delete the second paragraph (about the git repository), while keeping the first sentence (pointing people at the official releases).
